deps/v8/third_party/ittapi/src/ittnotify/LICENSE.GPL

```diff
@@ -0,0 +1,65 @@
+The GNU General Public License (GPL)
```

Comment:
Do we need to include this file ?  Since there is the option to distribute under the BSD or GPL licence I think we'd be opting for BSD and could not want to include this one as it could be flagged by licence scanners?